modules/apps/object/object-web/src/main/resources/META-INF/resources/js/modules/frontend-js-web.d.ts

```diff
@@ -27,4 +27,5 @@ declare module 'frontend-js-web' {
 		ok: boolean;
 		status: number;
 	}>;
+	const createResourceURL: any;
```

Comment:
Avoid using the `any` type, be as strict as possible.

If you have no idea about the type to use, you better start using unknown type instead:
```suggestion
	const createResourceURL: unknown;
```